lms/djangoapps/shoppingcart/migrations/0003_auto_20151217_0958.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('shoppingcart', '0002_auto_20151208_1034'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='courseregcodeitem',
+            name='mode',
+            field=models.SlugField(default=b'honor'),
+        ),
+        migrations.AlterField(
+            model_name='paidcourseregistration',
+            name='mode',
+            field=models.SlugField(default=b'honor'),
+        ),
+    ]
```

Comment:
I can only comment on this diff - I'm not familiar with the expiration_datetime changes up above. These tables are small in production: 12.5K rows for paidcourseregistration and 131 for courseregcodeitem.